src/lib/constants.ts

```diff
@@ -1,6 +1,8 @@
 export const BUILD_SCRIPT_URL =
   "https://raw.githubusercontent.com/Microsoft/bedrock/master/gitops/azure-devops/build.sh";
 
+export const HLD_PIPELINE_FILENAME = "manifest-generation.yaml";
```

Comment:
can we call this RENDER_HLD_PIPELINE_FILENAME instead? Feeling like it could be confused with the PROJECT_PIPELINE_FILENAME below